README.md

```diff
@@ -117,3 +117,11 @@ By default Ember table cell renderes simple text defined inside each `column`. T
   width: 180
 }
 ```
+## Migrating from old Ember table
+To support smooth migration from old version of Ember table (support only till ember 1.11), we have move the old source code to separate package [ember-table-legacy](https://github.com/Addepar/ember-table-legacy). It's a separate package from this Ember table package and you can install it using yarn or npm. This allows you to have 2 versions of ember table in your code base and you can start your migrating one table at at time. The recommended migration steps is as follow (if you are using ember 1.11):
```

Comment:
"The recommended migration steps is as follow" -> "The recommended migration steps _are_ as _follows_"